### ISSUE TITLE
.pairs with different email addresses does not seem to work

### ISSUE THREAD

**Issue by danieroux** [RoboPlugins/git-pair.idea#12]
This works well:

```
pairs:
    as: Alice Smith; alice
    bd: Bob Doe; bob

email:
    domain: example.com
```

The problem is that not all developers on the current project is from the same company, and have different email addresses. This, does not work:

```
pairs:
    as: Alice Smith
    bd: Bob Doe

email_addresses:
    as: alice.smith@example.com
    bd: bob.doe@another.corp
```

Does not work means: I can't click on the names to get tick marks next to the current pair.

**Comment by robert-wallis** [comment]
Great find!  The code is looking for a semicolon to figure out the email id. https://github.com/RoboPlugins/git-pair.idea/blob/6fe723299dc3945fbbbcb4d8711116e5ddf6c4de/src/gitpair/pairing/PairConfig.java#L70
When pairing across domains, I guess the domain should favor the first person like: `first+second@first.com` because GSuite email would be routed correctly to `first@first.com` in case something like CI wanted to email about a build failure.

In practice, our workaround is to use the client's domain, and make up fake emails.  For example if the client is `example.com`, I work for `genui.co` and I'm pairing with Alice, then the pair might look like `alice+robert-genui@example.com`.  And if I pair with Tim it might be `robert-genui+tim@example.com` which won't route email anywyere.  But I think supporting real email is a better fix.

**Comment by robert-wallis** [comment]
JetBrains needs a few days to manually verify the new version of the plugin.  If you can't wait until then, you can manually install the jar file from here: https://github.com/RoboPlugins/git-pair.idea/releases/download/v1.5/GitPair.jar

**Comment by danieroux** [comment]
Thank you for the quick turnaround!

**Comment by robert-wallis** [comment]
JetBrains has released 1.5!  That was quick. 🛃https://plugins.jetbrains.com/plugin/8135-git-pair
